libpod/oci.go

```diff
@@ -411,9 +415,10 @@ func (r *OCIRuntime) execContainer(c *Container, cmd, capAdd, env []string, tty
 		args = append(args, "--env", envVar)
 	}
 
-	// Append container ID and command
+	// Append container ID, name and command
 	args = append(args, c.ID())
 	args = append(args, cmd...)
+	args = append(args, c.Name())
```

Comment:
Wait, we need this for exec?

Reply:
actually we don't need this at all anymore, not even for create. good catch

Reply:
jk we want to conform to docker, but I'll remove it from exec, I think it was actually a rebase issue...